markdown/3.x/en/faq.md

```diff
@@ -20,7 +20,7 @@ You can add Angular's `ngNonBindable` attribute to the parent element to prevent
 ```
 The `<ng-container>` tag will not be rendered in the DOM.
 
-Alternatively, you can change the default FACT-Finder Web Components binding delimiters via `ff-communication`'s `mustache-delimiters` attribute. For more information please check the `Underlying Engine (Mustache)` section of the [Template Engine documentation](/documentation/3.x/template-engine). Please note that binding HTML still requires a single curly bracket, what will cause an Angular error. You can utilize `ngNonBindable` to handle this issue as well:
+Alternatively, you can change the default FACT-Finder Web Components binding delimiters via `ff-communication`'s `mustache-delimiters` attribute. For more information please check the `Underlying Engine (Mustache)` section of the [Template Engine documentation](/documentation/3.x/template-engine). Please note that binding HTML still requires an additional pair of (single) curly brackets, what will cause an Angular error. You can utilize `ngNonBindable` to handle this issue as well:
```

Comment:
Oops, I meant the `(single)` to be replaced by `single` or be removed altogether. I guess have to be more precise about what I actually mean.
Both work fine for me. You may choose.

Reply:
I just blindly committed your suggestion :D fixed